File: MY PROGRAMS/LB/JAVA/NUMBER/a2q2.java
/*	2. Accept one number from user and print that number of * on screen.	*/




import java.util.*;

class Assignment2
{
	public void Display(int ino)
	{
		if(ino<0)
		{
			ino=-ino;
		}

		while(ino>0)
		{
			System.out.print("*\t");

			ino--;
		}
	}
}

class a2q2
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);

		int ivalue=0;

		System.out.println("Enter the no.: ");
		ivalue=sobj.nextInt();

		Assignment2 aobj=new Assignment2();

		aobj.Display(ivalue);
	}
}

File: MY PROGRAMS/LB/CPP/SELECTION,SEQUENCE AND  ITERATION/a2q2.cpp
/* Accept one number from user and print that number of * on screen. */




#include<iostream>

using namespace std;

class Star
{
	public:
		void Display(int);
};

void Star::Display(int ino)
{
	int i=0;

	if(ino<0)
	{
		ino=-ino;
	}

	i=1;

	while(i<=ino)
	{
		cout<<" * ";

		i++;
	}
}

int main()
{
	int ivalue=0;

	cout<<"Enter the no.:";
	cin>>ivalue;

	Star sobj;

	sobj.Display(ivalue);

	return(0);
}





